src/sparseml/exporters/transforms/utils/matching.py

```diff
@@ -333,6 +333,15 @@ def _match_children(
         return False
 
     children = graph.get_node_children(node)
+    if children == [] and all(
+        all(op.startswith("Optional-") for op in ops) for ops in children_ops
```

Comment:
we should make `"Optional-"` a constant for consistency

Reply:
Good call, will do

Reply:
bump